### ISSUE TITLE
Scores no longer showing up

### ISSUE THREAD

**Issue by Idyllwilde** [ashercs/SpicetifyAOTYScores#9]
First of all, I've really been enjoying the Extension. Great work!

Is it possible that with the latest Spotify update, the extension isn't working anymore?

Spotify for Windows (64 bit)
1.2.26.1187.g36b715a1
Spicetify v2.28.1

**Comment by Rophiang** [comment]
The scores aren't showing up for me either with the latest Spotify updates. The refresh and delete buttons still show up and seemingly work but the scores aren't popping up.

Image:
![image](https://github.com/ashercs/SpicetifyAOTYScores/assets/68212612/0b1c8911-4e1c-4883-bae3-8bd8443b1ae6)

Spotify for Windows (64 bit)
1.2.26.1187.g36b715a1
Spicetify v2.29.1

**Comment by RetardedNConfused** [comment]
6 months

**Comment by OketZzz** [comment]
still not working ):

**Comment by GhosttRiderrr** [comment]
Yep still not working. Hope it gets fixed sometime, this feature would be awesome.

**Comment by x1yl** [comment]
This project seems to have been abandoned. I've forked this project [here](https://github.com/x1yl/BetterSpicetifyAOTYScores) and fixed the issue causing the scores to not show up. It would be amazing if people could test extension out.
